src/OSPSuite.Infrastructure.Import/Core/Mappers/DataSetToDataRepositoryMapper.cs

```diff
@@ -190,27 +193,16 @@ private static DataColumn findColumnByName(IEnumerable<DataColumn> columns, stri
             throw new ColumnNotFoundException(name);
          return column;
       }
-      private static void addExtendedPropertyForSource(string fileName, string sheetName,  DataRepository dataRepository)
+
+      private static void addExtendedPropertyForSource(string fileName, string sheetName, DataRepository dataRepository)
       {
-         if (!string.IsNullOrEmpty(fileName))
-         {
-            var sourceProperty =
-               Activator.CreateInstance(typeof(ExtendedProperty<>).MakeGenericType(fileName.GetType()))
-                  as IExtendedProperty;
-            if (sourceProperty != null)
```

Comment:
WTF. I have no idea what this code is doing so I removed it for something that makes sense
I commented the code out because I do not want to save the source anymore